File: scripts/windowcloseall.lua
-- Command: WindowCloseAll

function window_close_all_command ()
  -- NOTE: main() is run every time the command is started.
  --       Use it to reset variables so they are ready to go.
  if context == CONTEXT_MAIN then
    initCommand()
    clearSelection()
  end
  window_close_all()
  end_command()
}

--[[
[Tips]
ToolTip=Close &All
StatusTip=Close all the windows:  CLOSEALL

[Prompt]
Alias=CLOSEALL, WINDOWCLOSEALL
--]]

File: src/commands/windowcloseall.c
/* Command: windowcloseall */

#include "commands.h"

int window_close_all_cmd(State *state)
{
    /* NOTE: main() is run every time the command is started.
     *       Use it to reset variables so they are ready to go.
     */
    if (state->context == CONTEXT_MAIN) {
        init_command();
        clear_selection();
    }
    window_close_all();
    end_command();
    return 0;
}

